src/main/scala/temple/test/ProjectTester.scala

```diff
@@ -16,15 +16,15 @@ object ProjectTester {
     s"sh -c '$command'".!!
 
   /** Configure the infrastructure for testing */
-  private def performSetup(templefile: Templefile, generatedPath: String): EndpointConfig = {
+  private def performSetup(templefile: Templefile, generatedPath: String): ProjectConfig = {
     println("🍪 Spinning up Kubernetes infrastructure...")
     // Deploy Kubernetes
     exec(s"sh $generatedPath/deploy.sh")
 
     // Grab Kong's URL for future requests
     val urls   = exec("minikube service kong --url").split("\n")
```

Comment:
Can we destructure this into variables?

Reply:
How do you mean? One for the exec and one for the split?